wavelength_router/wavelength_router.proto

```diff
@@ -23,7 +23,7 @@ import "github.com/openconfig/gnoi/types/types.proto";
 option (types.gnoi_version) = "0.1.0";
```

Comment:
this version will need to be updated -- do we know if there are implementations of this RPC already?

Reply:
Nope, fortunately, we caught this early on. With some luck, there will be implementations in 2020 :)

Reply:
Version is now updated. Can you please take another look?